NEWS.md

```diff
@@ -72,6 +72,14 @@ New library features
   write the output to a stream rather than returning a string ([#48625]).
 * `sizehint!(s, n)` now supports an optional `shrink` argument to disable shrinking ([#51929]).
 * New function `Docs.hasdoc(module, symbol)` tells whether a name has a docstring ([#52139]).
+* Passing an IOBuffer as a stdout argument for Process spawn now works as
+  expected, synchronized with `wait` or `success`, so a `Base.BufferStream` is
+  no longer required there for correctness to avoid data-races ([#TBD]).
+* After a process exits, `closewrite` will no longer be automatically called on
+  the stream passed to it. Call `wait` on the process instead to ensure the
+  content is fully written, then call `closewrite` manually to avoid
+  data-races. Or use the callback form of `open` to have all that handled
```

Comment:
Does this mean [this advice](https://discourse.julialang.org/t/collecting-all-output-from-shell-commands/15592/7) is no longer valid, as `closewrite` is not called after `wait(p)`? Or this [code](https://github.com/JuliaPackaging/OutputCollectors.jl/blob/433285fbdca55f381902eb68737f8d3c247ab4dc/src/OutputCollectors.jl#L168)? I have not seen manual `closewrite` calls after `wait` calls ever so I am concerned this will be an unexpected change that will break working code.